ProcessLib/AnalyticalJacobianAssembler.h

```diff
@@ -34,6 +37,14 @@ class AnalyticalJacobianAssembler final : public AbstractJacobianAssembler
         const double dx_dx, std::vector<double>& local_M_data,
         std::vector<double>& local_K_data, std::vector<double>& local_b_data,
         std::vector<double>& local_Jac_data) override;
+
+    void assembleWithJacobianAndCoupling(
```

Comment:
docs missing.
Maybe it would be better to call this `assembleWithJacobianWeakCoupling` because "Coupling" is not a product of this function like "Jacobian".

Reply:
Docs was added and the name was changed as well.